tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/MsTestV2EvpTests.cs

```diff
@@ -40,11 +43,19 @@ public void SubmitTraces(string packageVersion)
             var testModules = new List<MockCIVisibilityTestModule>();
             var expectedTestCount = version.CompareTo(new Version("2.2.5")) < 0 ? 13 : 15;
 
+            var sessionId = SpanIdGenerator.CreateNew();
+            var sessionCommand = "test command";
+            var sessionWorkingDirectory = "C:\\evp_demo\\working_directory";
+            SetEnvironmentVariable(HttpHeaderNames.TraceId.Replace(".", "_").Replace("-", "_").ToUpperInvariant(), sessionId.ToString(CultureInfo.InvariantCulture));
+            SetEnvironmentVariable(HttpHeaderNames.ParentId.Replace(".", "_").Replace("-", "_").ToUpperInvariant(), sessionId.ToString(CultureInfo.InvariantCulture));
+            SetEnvironmentVariable(TestSuiteVisibilityTags.TestSessionCommandEnvironmentVariable, sessionCommand);
+            SetEnvironmentVariable(TestSuiteVisibilityTags.TestSessionWorkingDirectoryEnvironmentVariable, sessionWorkingDirectory);
+
             try
             {
-                SetEnvironmentVariable("DD_CIVISIBILITY_ENABLED", "1");
-                SetEnvironmentVariable("DD_TRACE_DEBUG", "0");
-                SetEnvironmentVariable("DD_DUMP_ILREWRITE_ENABLED", "0");
+                SetEnvironmentVariable(ConfigurationKeys.CIVisibility.Enabled, "1");
+                SetEnvironmentVariable(ConfigurationKeys.CIVisibility.ForceAgentsEvpProxy, "1");
+                SetEnvironmentVariable(ConfigurationKeys.DebugEnabled, "0");
```

Comment:
This is the default, any reason you have to set it explicitly?

Reply:
removed in [44a7b7d](https://github.com/DataDog/dd-trace-dotnet/pull/3389/commits/44a7b7d87f83fa43b88233fb52acbb295aad2d15)